File: scripts/fix-plants-id-jsx.py
from pathlib import Path

p = Path("app/plants/[id]/page.tsx")
text = p.read_text(encoding="utf-8")

start = text.find("          {/* Seller Details Section */}")
end = text.find("      {/* Checkout Preview Overlay */}")
if start == -1 or end == -1:
    raise SystemExit(f"markers not found start={start} end={end}")

replacement = """                )}

              </motion.div>
            </AnimatePresence>
          </div>
        </div>
      </div>

      """

new_text = text[:start] + replacement + text[end:]
p.write_text(new_text, encoding="utf-8")
print("patched seller section and closings")

File: scripts/fix-plants-id-jsx.mjs
import { readFileSync, writeFileSync } from 'fs';

const p = 'app/plants/[id]/page.tsx';
let text = readFileSync(p, 'utf8');

const start = text.indexOf('          {/* Seller Details Section */}');
const end = text.indexOf('      {/* Checkout Preview Overlay */}');
if (start === -1 || end === -1) {
  throw new Error(`markers not found start=${start} end=${end}`);
}

const replacement = `                )}

              </motion.div>
            </AnimatePresence>
          </div>
        </motion.div>
      </motion.div>

      `;

text = text.slice(0, start) + replacement + text.slice(end);
writeFileSync(p, text, 'utf8');
console.log('patched');
